lib/workers/repository/extract/index.spec.ts

```diff
@@ -42,10 +42,13 @@ describe('workers/repository/extract/index', () => {
     });
 
     it('warns if no packages found for a enabled manager', async () => {
-      config.enabledManagers = ['npm'];
+      config.enabledManagers = ['npm', 'custom.regex'];
       managerFiles.getManagerPackageFiles.mockResolvedValue([]);
       expect((await extractAllDependencies(config)).packageFiles).toEqual({});
-      expect(logger.debug).toHaveBeenCalled();
+      expect(logger.debug).toHaveBeenCalledWith(
+        { manager: 'regex' },
```

Comment:
This approach seems a bit confusing to me. It's a potential config error but we're being a bit fuzzy about the string used